### PR TITLE
Update featured.section Pulse #4

### PR DESCRIPTION
**Pull request by Lukewh**

### CODE REVIEW

File: prod/featured.section [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
-kgeography
-makemkv
-dosbox-x
+kstars
+element-desktop
+emacs
+cacher
+vidcutter
+ufw
+opendronemap
```
Comment (sergiusens) [inline prod/featured.section:8]: this one looks super interesting, it is new, but only on edge

File: prod/featured.section [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
+filebot
+opera
+konversation
+hyperion-ng
```
Comment (sergiusens) [inline prod/featured.section:12]: also new and only on edge

**Comment by Lukewh** [comment]
Discussed these two snaps with Holly, and she's happy for them to be featured. If she's happy, I'm happy! Are you happy @sergiusens ?

**Comment by shanepelletier** [comment]
Just chiming in to say I'm aware of this PR and will merge as soon as everyone's happy (assuming no one beats me to it).

**Comment by shanepelletier** [comment]
Merged and deployed to prod. Glad to join the happy party!

**Pull request merged**